### ISSUE TITLE
waterfox 56.0.4.1 (64-bit) archlinux, sometimes no button nor editor

### ISSUE THREAD

**Issue by sputnick-dev** [docwhat/itsalltext#99]
Hi,

I just installed waterfox right now to have IAT, but after installing it and restarting waterfox, I can set preferences, but I don't have any button, nor editor after right click on a textarea -> use default extension (or anything else).

On stackoverflow, I have this in the source, but on some other sites, I don't have the string `itsalltext` in the sources.

    <img src="chrome://itsalltext/locale/gumdrop.png" title="It's All Text!" style="cursor: pointer !important; display: none !important; position: absolute !important; padding: 0px !important; margin: 0px !important; border: medium none !important; width: 28px !important; height: 14px !important; opacity: 0.0152174 !important; left: 632px !important; top: 255px !important;">

**Comment by mb720** [comment]
I experience something similar on Waterfox (also 56.0.4.1):

There's no button and the shortcut for opening the editor doesn't work **until** I open Waterfox' add-ons manager.
After that, the add-on works.

Apart from this issue it would be great if this add-on could live on without Firefox since it's so useful.

**Comment by sputnick-dev** [comment]
Still no IAT after opening IAT preferences in addon manager, restarting waterfox...

**Comment by sputnick-dev** [comment]
Figured out how to work with IAT by restarting from scratch my profile (deleted/recreated).

Works now but only by example on stackoverflow, but not in gmail, sogo, github

**Comment by sputnick-dev** [comment]
Very unstable and not reliable now :/

**Comment by docwhat** [comment]
Sorry.  But IAT [is dead](https://github.com/docwhat/itsalltext/blob/master/README.md). :-(  

I don't plan on doing any more updates to this code base.

**Comment by mb720** [comment]
Thanks for all your work, it's a great add-on.

**Comment by sputnick-dev** [comment]
@docwhat: Yes, I agree @mb720 and I said thanks for your work, just a question: do you think IAT should work with waterfox or you don't know ?

**Comment by docwhat** [comment]
I have no idea. 😔

**Comment by sputnick-dev** [comment]
Found a solution by disabling multiprocess: https://www.reddit.com/r/waterfox/comments/826gft/itsalltext_addon_is_very_unstable_not_reliable/

**Comment by mb720** [comment]
> do you think IAT should work with waterfox or you don't know ?

I'm using IAT right now on Waterfox.

I can confirm that IAT works on Waterfox in a non-electrolysis (multi process) tab.

@docwhat: Given the current information about the breakage, where in IAT's code would you start looking to make it run without workarounds?

**Comment by docwhat** [comment]
Read the debugging document and check out the logs.
